coopaname_custom_contract/report/reports.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2020 Coop IT Easy
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <report id="action_report_hr_cae_cdi"
+            model="hr.contract"
+            print_report_name="'Contract - %s' % (object.name)"
```

Comment:
what is the difference between the `print_report_name` and `attachment` keys? I currently use the latter to automatically store as attachement upon printing.